### PR TITLE
UPDATE main.rs.

### PR DESCRIPTION
**Pull request by suleymanovdev**

### CODE REVIEW

**Comment by suleymanovdev** [comment]
More designed version.

File: src/main.rs [hunk 1]
```diff
@@ -11,9 +11,9 @@ impl Operation {
     fn evaluate(&self, num1: u32, num2 : u32) -> u32 {
         match self {
             Operation::Add  => num1 + num2, // returns num1 + num2 
-            Operation::Sub  => num1 - num2, // returns ... 
-            Operation::Mult => num1 * num2, // returns ...
-            Operation::Div  => num1 / num2, // returns ...
+            Operation::Sub  => num1 - num2, // returns num1 - num2
```
Comment (istadem2077) [inline src/main.rs:14]: Unneeded absolutely, redundant. DRY.

File: src/main.rs [hunk 2]
```diff
@@ -49,19 +49,19 @@ impl Input {
 }
 
 pub fn main() {
-    println!("TEXT-BASED CALCULATOR");
+    println!("TEXT-BASED CALCULATOR on RUST");
```
Comment (istadem2077) [inline src/main.rs:52]: Again redundant, it is pretty obvious that this program is written in Rust.

File: src/main.rs [hunk 3]
```diff
@@ -49,19 +49,19 @@ impl Input {
...
 
         let result : u32 =  operation.evaluate(num1, num2);
 
         println!("");
-        println!("The result of the calculation is : {}", result);
+        println!("The result of the calculation is :~> {} .", result);
     }
-}
\ No newline at end of file
+}
```
Comment (istadem2077) [inline src/main.rs:67]: Why.

**Review by istadem2077 — CHANGES_REQUESTED** [review]
Added main redundant comments(DRY), and unnecessary  changes.

**Pull request closed without merging**